wasmegg/rockets-tracker/src/components/LegendariesStudyOptInForm.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+    <div v-if="!preferenceOnFile" class="rounded-md bg-green-50 px-4 sm:px-6 py-4">
+      <h3 class="text-sm font-medium text-green-800">Anonymous study on legendary artifacts</h3>
+      <div class="mt-2 text-sm text-green-700">
+        <p>
+          An anonymous study is under way to understand how many legendary artifacts players tend to
+          possess at this point, and the relative frequency of different legendaries. If you enter the
+          study, a minimum amount of information is automatically submitted when your save is loaded.
+        </p>
+        <p v-if="!showDetails" class="mt-2 underline cursor-pointer" @click="showDetails = true">
+          Click here to learn more
+        </p>
+        <p v-else class="mt-2">
```

Comment:
This should probably be updated since it's not actually accurate with this change. It's sending the raw EID